gdx/src/com/badlogic/gdx/graphics/g3d/utils/FirstPersonCameraController.java

```diff
@@ -26,17 +26,20 @@
 /** Takes a {@link Camera} instance and controls it via w,a,s,d and mouse panning.
  * @author badlogic */
 public class FirstPersonCameraController extends InputAdapter {
-	private final Camera camera;

```

Comment:
could you please keep fields ordering? it's better for history/diffs

Reply:
done